gradle.properties

```diff
@@ -4,7 +4,7 @@ version=2.9.0-SNAPSHOT
 ballerinaLangVersion=2201.7.0-20230619-175900-bb4e4544
 ballerinaTomlParserVersion=1.2.2
 commonsLang3Version=3.8.1
-nettyVersion=4.1.86.Final
+nettyVersion=4.1.94.Final
 nettyTcnativeVersion=2.0.54.Final
```

Comment:
We also need to update the `nettyTcnativeVersion` to the latest version which is compatible with this netty version - `2.0.61.Final`